src/pipeline/parse.js

```diff
@@ -108,6 +108,49 @@ exports.getDataJS = (ast, DATA_DIR) => {
   return `module.exports = ${JSON.stringify(data)}`;
 }
 
+
+exports.getHighlightJS = (ast) => {
+
+  const languageMap = {
+    js: 'javascript'
```

Comment:
yeah exactly. if the language isn't in `languageMap` it will still get passed in as-is, so 

````
```javascript 
````

would still work